openedx/core/djangoapps/waffle_utils/testutils.py

```diff
@@ -0,0 +1,60 @@
+"""
+Test utilities for waffle utilities.
+"""
+
+from functools import wraps
+
+from waffle.testutils import override_flag
+
+
+def override_waffle_flag(flag, active):
```

Comment:
Maybe this is overkill, but I was wondering how to provide a course key to the override. It seems we need to have at least some tests that verify that the behavior works correctly with a course override in play.